src/main/cpp/util/InstallDirs.cpp

```diff
@@ -23,21 +23,21 @@
 #include "util/Errors.h"
```

Comment:
Why did you reformat this file? Did you add some functionality? If not, could you restore it please? That'll make it easier for Lynn to merge the upstream code into our repo.